<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gvobjwidget</class>
 <widget class="QMainWindow" name="gvobjwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gvobjwidget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>box</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>component</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ellipse</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>diamond</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>parallelogram</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mdiamond</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>underline</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cylinder</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>note</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>star</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>none</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tab</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>folder</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cds</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gray</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>indigo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>orange</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>navy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>steelblue</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>filled</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>shape</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>col style</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>200</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>250</y>
      <width>256</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialization</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>284</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
